.github/workflows/release-macos.yml

```diff
@@ -45,6 +45,11 @@ jobs:
                 rustup update --no-self-update stable
                 rustup default stable
 
+            - name: Set macOS 11 deployment target
+              run: |
+                echo "MACOSX_DEPLOYMENT_TARGET=11" >> $GITHUB_ENV
+                echo "RUSTFLAGS=-C link-args=-mmacosx-version-min=11" >> $GITHUB_ENV
```

Comment:
According to https://doc.rust-lang.org/rustc/platform-support/apple-darwin.html? this line is redundant, as rustc obeys the `MACOSX_DEPLOYMENT_TARGET` flag


```suggestion
```